models/_sector/nft/trades/ethereum/platforms/_schema.yml

```diff
@@ -0,0 +1,92 @@
+version: 2
+
+models:
+ - name: archipelago_ethereum_base_trades
```

Comment:
if we leave out column definitions in schema file(s), then we lose some flexibility to populate the docs page. i'm not sure how much usage there is there, but is something to consider.
maybe @0xBoxer can help understand the usage on spellbook docs page and if we remove columns here, then we could be stripping out details provided there

Reply:
definitely still the plan to add all columns in the schema file with proper descriptions and tests.

Big bonus is that we only have to specify the tests once and they'll be automatically applied to all platform models.